README.md

```diff
@@ -1,180 +1,113 @@
-
-To do that, you will need to `git clone` this repository's main branch following one of two ways.
-
-1. Open any command line shell that has been configured with git and execute the following command:
-```sh
-git clone https://github.com/Qiskit/qiskit-metal.git
-```
-2. Alternatively, you can download and use the user interface [GitHub Desktop GUI](https://desktop.github.com/) and refer to these [notes](https://help.github.com/en/desktop/contributing-to-projects/cloning-a-repository-from-github-to-github-desktop).
-
-Now that you have a local copy of the code, you can install Qiskit Metal either in a virtual [conda environment](https://docs.conda.io/en/latest/miniconda.html) or in a virtual Python environment, as described below. We recommend conda.
-
-Notes:
+It is also recommended that you make a new conda environment before installing:
 
-* For your own sanity, it is recommended to read this document in its entirety before proceeding.
-* On Windows, the conda environment is strongly recommended because Shapely is difficult to install directly via pip.
-
-### Setup in a conda environment (preferred setup)
-
-If you did not yet install conda, please follow these [instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/).
-We will setup a conda environment to use the local copy of Qiskit Metal you obtained in the previous section. This approach enables you to immediately observe the effect of your code modifications.
-
-For this section you will need to use the command line. If you use github desktop, you can open one from the menu `Repository -> Open In....`
-
-#### Option 1: A new environment
-The most reliable way to set up a qiskit_metal environment is to build one from scratch using the provided conda environment specification file `environment.yml`.
-To do so, first navigate to the folder created by the clone. For example:
-```sh
-cd qiskit-metal
-```
-Once you are in the folder that contains the `environemnt.yml` file, execute the following installation commands:
-```sh
+```bash
 conda env create -n <env_name> environment.yml
 conda activate <env_name>
 python -m pip install --no-deps -e .
 ```
-Note the use of `--no-deps`. Indeed the `environment.yml` already instructs conda to install all the necessary package dependencies. We therefore prevent `setup.py` from overwriting them with the pip-equivalent packages, which might not be compatible with conda. 
 
-This creates a new environment with name `<env_name>` with all the necessary library dependencies.
-Then it activates the new environment.
-Finally installs the local Qiskit Metal code inside that environment.
+To edit the code and truly customize the experience, if PyPI gives you trouble, or to install from source, follow the instructions in the [documentation](https://qiskit.org/documentation/metal/installation.html) and/or the [installation instructions for developers](/README_developers.md).
```

Comment:
Maybe adding a little bold heading here would be nice, to show this is the third option, it's a little bit easy to miss the developer option